tools/c7n_tencentcloud/requirements.txt

```diff
@@ -5,7 +5,7 @@ crcmod==1.7 ; python_version >= "3.7" and python_version < "4.0"
 idna==3.4 ; python_version >= "3.7" and python_version < "4.0"
 pycryptodome==3.16.0 ; python_version >= "3.7" and python_version < "4.0"
 pytz==2022.6 ; python_version >= "3.7" and python_version < "4.0"
-requests==2.27.1 ; python_version >= "3.7" and python_version < "4.0"
+requests==2.28.1 ; python_version >= "3.7" and python_version < "4"
```

Comment:
requirements.txt is generated, not manually edited. additionally, the cos-python-sdk-v5 package has a restriction on requests which needs to be loosened